components/bytes/src/lib.rs

```diff
@@ -245,6 +251,20 @@ impl<'a, B: MemBufMut> Write for Writer<'a, B> {
     }
 }
 
+impl Write for &mut dyn MemBufMut {
```

Comment:
&mut dyn MemBufMut can not be wrapped to Writer, because the size of dyn MemBufMut can not know at compile time.
So I directly implement Write for &mut dyn MemBufMut.